storage/scylla_api.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLA_API_H_
+#define TA_SCYLLA_API_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "accelerator/common_core.h"
+#include "cassandra.h"
+#include "scylla_config.h"
+
+typedef struct scylla_iota_transaction_s scylla_iota_transaction_t;
```

Comment:
We can do typedef in where this struct is defined

Reply:
But than only scylla_api.c could use it

Reply:
so we could define the object here. It is more similar to our coding style as well

Reply:
The link is wrong one

Reply:
leave this one and don't resovle it

Reply:
got it